src/components/dialogs/ConfirmDialog.vue

```diff
@@ -5,6 +5,7 @@
     :max-width="options.width"
     :style="{ zIndex: options.zIndex }"
     :persistent="options.persistent"
+    :attach="attach"
```

Comment:
This is so that this dialog is mounted within our component-under-test, so we can look for it in the DOM (wrapper). Otherwise the dialog is mounted at the root level of the DOM (which we don't have access since we only  mount/wrap our component).

Reply:
Right on, beat me to it :)